cmd/backup.go

```diff
@@ -13,20 +13,54 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package cmd
 
 import (
-	"fmt"
+	"github.com/cluetec/lifeboat/internal/config"
+	"github.com/cluetec/lifeboat/internal/destination"
+	"github.com/cluetec/lifeboat/internal/logging"
+	"github.com/cluetec/lifeboat/internal/source"
+	"log/slog"
+	"os"
 
 	"github.com/spf13/cobra"
 )
 
+var globalConfig *config.Config
```

Comment:
Why do we need this to be a global variable? This will make the command harder to test. How about something like a `getConfig` function which gets called in `Run` instead of the bootstrapping in `init()`

Reply:
Mhh, I'm currently thinking, that we also could move the whole `init` method into the `Run` method.
In the documentation the `init` method only contains adding subcommands to the cobra command and the mapping for flags between cobra and viper. So I guess it would make sense to stick to that approach.

WDYT?

Reply:
That is essentially what I proposed but just extracting it into a function. So yes, sounds reasonable

Reply:
done :-) I moved the `InitViper` call into the `config.New` method to reduce the overhead int he `Run` method. The only think where I'm not sure if this is okay, or if we should find a better solution is the `logging.InitSlog(c.GetLogLevel())` in the `Run` method.

Any preferences?

Reply:
I think we can leave it as it is for now.